src/transpose.c

```diff
@@ -18,23 +18,28 @@ SEXP transpose(SEXP l, SEXP fill, SEXP ignoreArg, SEXP keepNamesArg) {
   if (length(fill) != 1)
     error(_("fill must be a length 1 vector, such as the default NA"));
   R_len_t ln = LENGTH(l);
+  if (!isLogical(listColsArg) || LOGICAL(listColsArg)[0]==NA_LOGICAL)
```

Comment:
```suggestion
  if (!IS_TRUE_OR_FALSE(listColsArg))
```

Reply:
See this and some other comments marked resolved without being addressed, is there maybe an unpublished commit or review?